build_info/haydenfetch.control

```diff
@@ -0,0 +1,8 @@
+Package: haydenfetch
+Architecture: @DEB_ARCH@
+Version: @DEB_HAYDENFETCH_V@
+Depends: jp2a, coreutils, neofetch
```

Comment:
``coreutils`` should instead be in the ``Pre-Depends`` field, since it's a core package